Makefile

```diff
@@ -1,8 +1,8 @@
 unit:
-	node_modules/.bin/cucumber-js --tags "@unit.offline or @unit.algod or @unit.indexer or @unit.rekey" tests/cucumber/features --require tests/cucumber/steps/*
+	node_modules/.bin/cucumber-js --tags "@unit.applications" tests/cucumber/features --require tests/cucumber/steps/*
```

Comment:
The new client tests are in `@unit.responses`

Reply:
https://github.com/algorand/algorand-sdk-testing/blob/develop/features/unit/responses.feature

There is a new `the parsed response should equal the mock response.` that will be tricky. js/python shouldn't be too bad, but I'm a bit worried about how go will be.